exercises/grains/.meta/tests.toml

```diff
@@ -0,0 +1,35 @@
+[canonical-tests]
+
+# Square returns the number of grains on 1 square
+"9fbde8de-36b2-49de-baf2-cd42d6f28405" = true
+
+# Square returns the number of grains on 2 squares
+"ee1f30c2-01d8-4298-b25d-c677331b5e6d" = true
+
+# Square returns the number of grains on 3 squares
+"10f45584-2fc3-4875-8ec6-666065d1163b" = true
+
+# Square returns the number of grains on 4 squares
+"a7cbe01b-36f4-4601-b053-c5f6ae055170" = true
+
+# Square returns the number of grains on 16 squares
+"c50acc89-8535-44e4-918f-b848ad2817d4" = true
+
+# Square returns the number of grains on 32 squares
+"acd81b46-c2ad-4951-b848-80d15ed5a04f" = true
+
+# Square returns the number of grains on 64 squares
+"c73b470a-5efb-4d53-9ac6-c5f6487f227b" = true
+
+# Square raises an exception for 0 squares
+"1d47d832-3e85-4974-9466-5bd35af484e3" = true
+
+# Square raises an exception for negative squares
+"61974483-eeb2-465e-be54-ca5dde366453" = true
+
+# Square raises an exception for greater than 64 squares
+"a95e4374-f32c-45a7-a10d-ffec475c012f" = true
+
+# Total returns the total number of grains on the board
+"6eb07385-3659-4b45-a6be-9dc474222750" = true
+
```

Comment:
Replaced with what `canonical_data_syncer` would have output:

```suggestion
[canonical-tests]

# 1
"9fbde8de-36b2-49de-baf2-cd42d6f28405" = true

# 2
"ee1f30c2-01d8-4298-b25d-c677331b5e6d" = true

# 3
"10f45584-2fc3-4875-8ec6-666065d1163b" = true

# 4
"a7cbe01b-36f4-4601-b053-c5f6ae055170" = true

# 16
"c50acc89-8535-44e4-918f-b848ad2817d4" = true

# 32
"acd81b46-c2ad-4951-b848-80d15ed5a04f" = true

# 64
"c73b470a-5efb-4d53-9ac6-c5f6487f227b" = true

# square 0 raises an exception
"1d47d832-3e85-4974-9466-5bd35af484e3" = true

# negative square raises an exception
"61974483-eeb2-465e-be54-ca5dde366453" = true

# square greater than 64 raises an exception
"a95e4374-f32c-45a7-a10d-ffec475c012f" = true

# returns the total number of grains on the board
"6eb07385-3659-4b45-a6be-9dc474222750" = true
```

Reply:
Yeesh. But OK, changes to make better names should happen in the spec, as you suggest on #1674